test/src/_assets/scss/components/_c-test-case.scss

```diff
@@ -109,13 +109,12 @@ $c-test-case-test-subject-height: 300;
 
     // Inner grid item including label
     .l-grid__item > div {
-        align-items: center;
         background-color: $g-color-test-case-subject-background;
-        display: flex;
         font-family: $shell-g-font-family-monospace;
         font-weight: normal;
-        height: $c-test-case-test-subject-height * 1px;
+        padding: 1rem;
```

Comment:
Can do, I saw you'd declared the height in `px` so didn't realise we were writing these for scales, just chucked some arbitrary padding in